### PR TITLE
[13] feat!: remove tables concept

### PR DESCRIPTION
**Pull request by v-rocheleau**

### CODE REVIEW

File: chord_metadata_service/chord/urls.py [hunk 1]
```diff
@@ -20,13 +21,22 @@
...
+         name="chord-dataset-detail"),
+    path('datasets/<str:dataset_id>/summary', views_search.dataset_summary, name="chord-dataset-summary"),
+    path('datasets/<str:dataset_id>/data-types/<str:data_type>', views_data_types.dataset_data_type,
+         name="chord-dataset-data-type"),
+
+    path('datasets/<str:dataset_id>/data-types', views_data_types.dataset_datatype_summary,
+         name="chord-dataset-data-type-summary"),
+
+    path('datasets/<str:dataset_id>/search', views_search.public_dataset_search, name="public-dataset-search"),
+    path('private/datasets/<str:dataset_id>/search', views_search.private_dataset_search,
```
Comment (davidlougheed) [inline chord_metadata_service/chord/urls.py:41]: would be nice to deprecate dataset search vs private/datasets search (although maybe thats a v14 thing)

Comment (davidlougheed) [reply chord_metadata_service/chord/urls.py:40]: also how does this search actually work, given that we dont have one data type per dataset?

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: chord_metadata_service/chord/urls.py (hunk 1 above)
Comment (v-rocheleau) [reply chord_metadata_service/chord/urls.py:41]: The data_type is given by a query parameter extracted by  `views_searh.get_chord_search_parameters`.

File: chord_metadata_service/chord/models.py [hunk 2]
```diff
@@ -169,47 +169,47 @@ def __str__(self):
...
+#         return f"{self.dataset} -> {self.table_id}"
 
 
-class Table(models.Model):
-    TABLE_DATA_TYPE_CHOICES = (
-        (DATA_TYPE_EXPERIMENT, DATA_TYPE_EXPERIMENT),
-        (DATA_TYPE_PHENOPACKET, DATA_TYPE_PHENOPACKET),
-        (DATA_TYPE_MCODEPACKET, DATA_TYPE_MCODEPACKET),
-    )
+# class Table(models.Model):
```
Comment (davidlougheed) [inline chord_metadata_service/chord/models.py]: remove maybe?

Comment (v-rocheleau) [reply chord_metadata_service/chord/models.py]: Oups yeah

File: chord_metadata_service/metadata/settings.py [hunk 3]
```diff
@@ -377,4 +377,8 @@ def get_secret(path):
 # SPECTACULAR_SETTINGS['SERVERS'] defines the url to which calls are made when
 # testing a request within the swagger UI
 if CHORD_URL:
-    SPECTACULAR_SETTINGS['SERVERS'] = [{'url': CHORD_URL + FORCE_SCRIPT_NAME}]
+    SPECTACULAR_SETTINGS['SERVERS'] = [
+        # {'url': CHORD_URL + FORCE_SCRIPT_NAME},
+        {'url': 'http://localhost:8000'}
+    ]
+print(SPECTACULAR_SETTINGS)
```
Comment (davidlougheed) [inline chord_metadata_service/metadata/settings.py]: rm

**Review by davidlougheed — CHANGES_REQUESTED** [review]

**Review by davidlougheed — APPROVED** [review]
lgtm

**Pull request merged**